pages/_app.tsx

```diff
@@ -1,6 +1,36 @@
-import '@/styles/globals.css'
-import type { AppProps } from 'next/app'
+import ButtonCta from "@/src/components/Cta/buttonCta";
+import "/styles/globals.scss";
+import {ChangeEvent, useState} from "react";
+import Input from "@/src/components/input/input";
 
-export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+function App() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleEmailChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setEmail(e.target.value);
+  };
+
+  const handlePasswordChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.target.value);
+  };
+  const test: any = {
+    display: "flex",
+    gap: "30px",
+    flexDirection: "column",
+    margin: "20px",
+  };
```

Comment:
any 타입을 따로 잡아줄 필요는 없을 것 같구요~
```suggestion
  const test = {
    display: "flex",
    gap: "30px",
    flexDirection: "column",
    margin: "20px",
  };
```

Reply:
상수값들은 컴포넌트 내부에 위치하면 리렌더링될때마다 값이 계속 생성되기 때문에 컴포넌트 외부로 빼주시는게 좋습니다~

Reply:
test라는 이름을 보면 테스트 용도로 사용해주시고 삭제하지 않은 변수인 것 같기도 하네요!